### PR TITLE
"Hello world" project with two apps - Back-end

### PR DESCRIPTION
**Pull request by williamscch**
Create a new Rails API app called 'hello-rails-back-end'.
Initialize your project with Git.
Make sure that your project has a Postgres database set up.
Create a table for storing your messages and create 5 different greetings.
Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

[Link to react repo](https://github.com/williamscch/hello-react-front-end/pull/1)

### CODE REVIEW

**Review by tresorsawasawa — APPROVED** [review]
Hi 
This is here by error. I clicked the wrong button to submit the review.

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @williamscch  :wave: :wave: 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

You have submitted a project that where one of the repo doesn't use the updated Microverse README template and the other one doesn't any README. Make sure you add the correct **[README](https://github.com/microverseinc/readme-template)** on both repositories and document each one professionally.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @williamscch  :wave: :wave: 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

You have submitted a project where both of your repo doesn't use the updated Microverse README template. Make sure you add the correct **[README](https://github.com/microverseinc/readme-template)** on both repositories and document each one professionally.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by williamscch** [comment]
> # STATUS: INVALID CODE REVIEW REQUEST  
> Hi @williamscch wave wave While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request  
> You have submitted a project where both of your repo doesn't use the updated Microverse README template. Make sure you add the correct **[README](https://github.com/microverseinc/readme-template)** on both repositories and document each one professionally.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> _Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._
> 
> _Invalid Code Review Request **Does not count** into the code reviews limit._

Please be more specific, both repos has readme files well documented @tresorsawasawa

**Comment by williamscch** [comment]
> > # STATUS: INVALID CODE REVIEW REQUEST
> > Hi @williamscch wave wave While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> > ## Invalid Code Review Request
> > You have submitted a project where both of your repo doesn't use the updated Microverse README template. Make sure you add the correct **[README](https://github.com/microverseinc/readme-template)** on both repositories and document each one professionally.
> > Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> > Cheers and Happy coding!clapclapclap
> > _Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._
> > _Invalid Code Review Request **Does not count** into the code reviews limit._
> 
> Please be more specific, both repos has readme files well documented

Can you leave a link to that updated readme file you are talking about to check it @tresorsawasawa

**Comment by tresorsawasawa** [comment]
Link left on slack

File: app/models/greeting.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Greeting < ApplicationRecord
+end
```
Comment (tresorsawasawa) [inline app/models/greeting.rb:2]: - [x] Good job with your backend application, everything is working as expected. The issue is that your frontend app doesn't display any message. You can get **[here](https://github.com/williamscch/hello-react-front-end/pull/1/)** all comments reviews you need to fix for **[the frontend repo pull request](https://github.com/williamscch/hello-react-front-end/pull/1/)**

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @williamscch 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @williamscch 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**